script/checkout.py

```diff
@@ -45,6 +45,10 @@ def main():
   else:
     subprocess.check_call(["python3", "tools/git-sync-deps"])
 
+  # fetch ninja
+  print("> Fetching ninja")
+  subprocess.check_call(["bin/fetch-ninja"])
```

Comment:
For some reason, this script is missing on Windows, and there already is a `ninja.exe` binary in the `depot_tools` here. I'll see if adjusting the script accordingly helps!

Reply:
Ah, it's recommended on Windows to download a different `depot_tools` bundle instead of checking out the repo: https://storage.googleapis.com/chrome-infra/depot_tools.zip I'll set it up like that